### PR TITLE
fixed scroll styles (#297)

### PR DESCRIPTION
**Pull request by siddhant1**
ref #297

### CODE REVIEW

File: src/components/signal-viewer/index.css [hunk 1]
```diff
@@ -2,6 +2,7 @@
   padding: 10px 20px 15px 20px;
   width: fit-content;
   max-width: 100%;
+  height: 90%;
```
Comment (domoritz) [inline src/components/signal-viewer/index.css]: Why is this the right fix?

**Comment by siddhant1** [comment]
Giving the section in which the table is present a height kicks in the
overflow:scroll after that height

**Comment by domoritz** [comment]
Why is it exactly 90%? It seems so arbitrary.

**Comment by siddhant1** [comment]
Tried 100% to take the whole width of the upper wrapper , felt that it should be 10% smaller . I can use pixels but it wouldn't be responsive

**Comment by domoritz** [comment]
You can use 100% and padding, no?

**Comment by siddhant1** [comment]
Sure , let me have a look

**Comment by siddhant1** [comment]
Actually 100% is okay too

**Pull request merged**